uh_internal/apps/dailyduties/admin.py

```diff
@@ -11,5 +11,10 @@ class EmailPermalinkAdmin(admin.ModelAdmin):
     list_display = ('sender_name', 'sender_email', 'subject', 'slug')
 
 
+class EmailViewingRecordAdmin(admin.ModelAdmin):
+    list_display = ('user', 'mailbox', 'uid', 'replying', 'expiry_time')
+
+
 admin.site.register(DailyDuties, DailyDutiesAdmin)
 admin.site.register(EmailPermalink, EmailPermalinkAdmin)
```

Comment:
I think this line should have actually been removed. Test it to make sure, but I think it is no longer necessary.